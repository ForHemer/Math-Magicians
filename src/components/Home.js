function Home() {
  const par = 'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nulla et est tortor. Pellentesque justo eros, fermentum tempus nisi sed, malesuada scelerisque ligula. Nulla et vulputate nulla. Integer viverra ligula sapien. Donec non eros at justo interdum tempus.';

  return (
    <section className="Home">
      <h1>Welcome to our Page!</h1>
      <p>{par}</p>
      <p>{par}</p>
    </section>
  );
}

export default Home;
