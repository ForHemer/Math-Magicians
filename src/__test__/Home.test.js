import React from 'react';
import renderer from 'react-test-renderer';
import Home from '../components/Home';

test('renders correctly', () => {
  const home = renderer.create(<Home />).toJSON();
  expect(home).toMatchSnapshot();
});
