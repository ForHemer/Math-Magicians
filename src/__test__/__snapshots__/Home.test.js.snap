// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<section
  className="Home"
>
  <h1>
    Welcome to our Page!
  </h1>
  <p>
    Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nulla et est tortor. Pellentesque justo eros, fermentum tempus nisi sed, malesuada scelerisque ligula. Nulla et vulputate nulla. Integer viverra ligula sapien. Donec non eros at justo interdum tempus.
  </p>
  <p>
    Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nulla et est tortor. Pellentesque justo eros, fermentum tempus nisi sed, malesuada scelerisque ligula. Nulla et vulputate nulla. Integer viverra ligula sapien. Donec non eros at justo interdum tempus.
  </p>
</section>
`;
